sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -1054,12 +1116,12 @@ public Node visitArraySubquery(SqlBaseParser.ArraySubqueryContext ctx) {
     }
 
     /*
-    * case sensitivity like it is in postgres
-    * see also http://www.thenextage.com/wordpress/postgresql-case-sensitivity-part-1-the-ddl/
-    *
-    * unfortunately this has to be done in the parser because afterwards the
-    * knowledge of the IDENT / QUOTED_IDENT difference is lost
-    */
+     * case sensitivity like it is in postgres
+     * see also http://www.thenextage.com/wordpress/postgresql-case-sensitivity-part-1-the-ddl/
+     *
```

Comment:
Please avoid unrelated formatting changes. (Especially on a PR against a feature branch).

Maybe you can do those separate in a PR against master?